src/framing.c

```diff
@@ -1600,7 +1616,7 @@ void test_pack(const int *pl, const int **headers, int byteskip,
               ogg_stream_packetout(&os_de,&op_de); /* just catching them all */
```

Comment:
I didn't do it this way because I wasn't 100% sure it would be correct, eg. maybe the compiler could write uninitialised data back into the hole or maybe libogg could memcpy another stack-allocated ogg_packet into the struct (I haven't read the rest of the code yet.)

The memset() solution passes ``make check`` though and it's simpler. I guess it's up to you guys which one you prefer.